app/controllers/accountants_controller.rb

```diff
@@ -1,5 +1,4 @@
 class AccountantsController < ApplicationController
   def index
-    @user = current_user
```

Comment:
better add a #TODO comment here to let us know what you will do in future

Reply:
I am accessing partial views through this function of users